spec/rubocop/target_finder_spec.rb

```diff
@@ -510,18 +510,16 @@
     end
 
     it 'can exclude symlinks as well as directories' do
-      Dir.mktmpdir do |tmpdir|
-        create_empty_file(File.join(tmpdir, 'ruby5.rb'))
-        create_link('link', tmpdir)
+      create_empty_file(File.join(Dir.home, 'ruby5.rb'))
+      create_link('link', Dir.home)
```

Comment:
Using Dir.home instead of a second tmpdir, which maintains isolation while staying consistent with the test's intention.